### PR TITLE
Indebanvdhamer/performance

### PR DESCRIPTION
**Pull request by indebanvdhamer**
## Motivation

Webpack has pretty bad performance when it comes to the `scss-loader`. This PR improves performance by removing some of the default options, since they're not always needed. They can still be passed via the `.webpacker.js` file.

## Changes

- Improve (default) performance for the postcss loader by removing `postcss-preset-env`.
- Add support for the lodash treeshaking plugin to decrease bundle sizes

### CODE REVIEW

File: loaders/scss.js [hunk 1]
```diff
@@ -15,16 +16,11 @@ module.exports = (
           localIdentName: '[path]___[name]__[local]',
         }
       },
-      postcss({postcssPresetEnvOptions}),
+      postcss(postcssOpts),
       {
         loader: 'sass-loader',
         options: {
-          data: `
```
Comment (renanborgez) [inline loaders/scss.js]: 👍

File: plugins/lodash.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+const LodashModuleReplacementPlugin = require('lodash-webpack-plugin');
```
Comment (Amri91) [inline plugins/lodash.js]: I am not sure how useful this file is.

Can we do
```js
module.exports = require('lodash-webpack-plugin')
```
?

Comment (indebanvdhamer) [reply plugins/lodash.js]: We could. I was just trying to keep consistent with the file structure of the other plugins. Let's discuss.

**Comment by Amri91** [comment]
This is a breaking change, should be increase the version in the same merge to master?

File: plugins/lodash.js (hunk 2 above)
Comment (Amri91) [reply plugins/lodash.js]: I see, that is reasonable.

**Comment by renanborgez** [comment]
> This is a breaking change, should be increase the version in the same merge to master?

Yes

**Pull request merged**